examples/using-typescript/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ChangeEvent, forwardRef } from 'react';
+import { view as v, store as s } from 'react-easy-state';
+
+const store = s({
+  name: 'Peter',
+  setNameEvent(ev: ChangeEvent<HTMLInputElement>) {
+    if (ev.target) {
+      this.name = ev.target.value;
+    }
+  },
+});
+
+// properly retain these props for the views
+type AppProps = {
+  greeting: string;
+};
+
+const App = v(
+  forwardRef<HTMLLabelElement, AppProps>(
```

Comment:
Is this needed here or only used to test forwardRef with TS? In the latter case I think we should move this check under tests (it will confuse beginners who use this code as a reference).